packages/website/docs/Future-Features.md

```diff
@@ -79,3 +79,32 @@ Right now, Monte Carlo simulations are totally random. It would be nicer to be a
 
 ## Fixes
```

Comment:
can this be more clear than the word `Fixes` like something to convey that it's something we're currently unhappy with, right? like `upcoming fixes`?